osf-cli/pom.xml

```diff
@@ -128,6 +128,11 @@
             <artifactId>osf-packager</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
```

Comment:
In the past we have very loosely applied semantic versioning, X.Y.Z, where incrementing Z would be a drop-in (no breaking changes) replacement.  Incrementing Y would happen when binary compatibility breaks, and X when an API changes.

Because our code doesn't always have a clearly defined API, we don't follow this versioning strictly.  Probably after this refactoring we would bump the version up to 1.1.0, or 2.0.0.

With some refactoring coming down the pike (the work I've been doing to merge the different OSF-related packaging GitHub repositories) it will become more clear what modules are versioned together.